### PR TITLE
IBX-8229: Screenshots update in Dev-doc

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | (https://issues.ibexa.co/browse/IBX-8229)
| Edition       | master

Screenshots update in Dev-doc

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Redirects cover removed/moved pages
- [x] Code samples are working
- [x] PHP code samples have been fixed with PHP CS fixer
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/administration/img/admin_panel_assignments.png
Comment (adriendupuis) [inline docs/administration/img/admin_panel_assignments.png:1]: I would cut out in the middle to reduce width.
https://ez-systems-developer-documentation--2487.com.readthedocs.build/en/2487/administration/admin_panel/roles_admin_panel/

File: docs/administration/img/admin_panel_policies.png
Comment (adriendupuis) [inline docs/administration/img/admin_panel_policies.png:1]: Idem, I would reduce width.

**Review by adriendupuis — COMMENTED** [review]
Optionnally, few images' width could be reduced by cutting out middle emptiness.

I love improvement like delete_confirmation.png in [content_management_guide.md](https://ez-systems-developer-documentation--2487.com.readthedocs.build/en/2487/content_management/content_management_guide/#content-arrangement) or notification3.png in [notifications.md](https://ez-systems-developer-documentation--2487.com.readthedocs.build/en/2487/administration/back_office/notifications/)

File: docs/content_management/img/ckeditor-special-characters_arrows-on-top.png
Comment (mnocon) [inline docs/content_management/img/ckeditor-special-characters_arrows-on-top.png:1]: Same here - I think the original image was more readable, the new one seems blurry

File: docs/content_management/img/create_page.png
Comment (mnocon) [inline docs/content_management/img/create_page.png:1]: You're changing from letters to numbers - so the procedure should be updated to to use numbers as well (https://ez-systems-developer-documentation--2487.com.readthedocs.build/en/2487/content_management/pages/page_builder_guide/#what-is-page)

**Review by mnocon — COMMENTED** [review]
You've put a lot of effort into this and it shows 😄 

I have a couple of small suggestions, but it's great - thank you for taking care of this.

**Review by adriendupuis — APPROVED** [review]

File: docs/commerce/checkout/img/reorder_button.png
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/commerce/checkout/img/reorder_button.png:1]: Yes, good point. There was some technical issue while adding a frame.

**Pull request merged**